rules/config

```diff
@@ -178,7 +178,7 @@ INCLUDE_ROUTER_ADVERTISER ?= y
 
 # INCLUDE_KUBERNETES - if set to y kubernetes packages are installed to be able to
 # run as worker node in kubernetes cluster.
-INCLUDE_KUBERNETES ?= n
+INCLUDE_KUBERNETES ?= y
```

Comment:
is that for test purpose ?

Reply:
Yes, will change it back to "n" after test